firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests/L10nSuite2SnapshotTests.swift

```diff
@@ -184,11 +185,16 @@ class L10nSuite2SnapshotTests: L10nBaseSnapshotTests {
         // Tap on each option in that menu
         mozWaitForElementToExist(app.buttons["Shopping.ReviewQualityCard.ExpandButton"])
         app.buttons["Shopping.ReviewQualityCard.ExpandButton"].tap()
+        app.swipeUp(velocity: 15)
```

Comment:
The screenshot could be nicer to show more info on the quality.
![iPhone 14-ReviewQualityCard](https://github.com/mozilla-mobile/firefox-ios/assets/1740517/45f0d9ff-7048-4a2d-a6f9-3f8c35c8a424)

Reply:
Nit: We should take an screenshot before and after doing the swipe.

Reply:
Done! 😄